lib/cable_ready/operation_builder.rb

```diff
@@ -43,7 +43,10 @@ def add_operation_method(name)
           options.each { |key, value| options[key] = value.send("to_#{key}".to_sym) if value.respond_to?("to_#{key}".to_sym) }
         end
         options["selector"] = selector if selector && options.exclude?("selector")
-        options["selector"] = previous_selector if previous_selector && options.exclude?("selector")
+        if previous_selector && options.exclude?("selector")
+          options["selector"] = previous_selector
+          options["xpath"] = true if options["selector"].to_s.starts_with? "/"
```

Comment:
I'm wondering if it would make sense to also store the `xpath` option in a `previous_xpath` variable instead of basing it off of the `options["selector"]` value.

And even more meta, now that I'm thinking about it: couldn't we just leave out this `xpath` option all-together for all operations and check on the client-side if any given selector starts with a `/`? Regular CSS selectors shouldn't start with a slash.

Reply:
Maybe both? It feels a bit weird if the server side is building operations that aren't valid and passing them to the client anyway?

Reply:
Why wouldn't they be valid? 

Maybe we can keep it simple for now and just store the previous xpath value and leave it at. 

But it might be something to think about down the road.

Reply:
I dropped the last commit :ok_hand:

Reply:
Awesome, thank you!